versioned_docs/version-6.x/header-buttons.md

```diff
@@ -35,7 +35,7 @@ function StackScreen() {
 }
 ```
 
-![Header button](/assets/headers/header-button.png)
+<img src="/assets/7.x/fundamentals/buttonInHeader.png" width="500" alt="Button right in the header" />
```

Comment:
I see this logo isn't rounded as in `Configuring the header bar` section 😄 can we make it rounded?